package.py

```diff
@@ -836,6 +836,7 @@ def install_pip_requirements(query, requirements_file):
     runtime = query.runtime
     artifacts_dir = query.artifacts_dir
     docker = query.docker
+    temp_dir = query.temp_dir
```

Comment:
Isn't this `temp_dir` shadowed by `with tempdir(tmp_dir) as temp_dir` at line [872](https://github.com/terraform-aws-modules/terraform-aws-lambda/pull/230/files#diff-34b2a63c4b18a635042766329eca28d8801699eecf5e93eaedaeda1309931381L872)?

Reply:
I believe this may have been left over from a first pass I did. I will remove.

Reply:
```suggestion
```